examples/dns/main.tf

```diff
@@ -12,10 +44,11 @@ module "dns_basic" {
 
   resource_packs_gcp_url = var.resource_packs_gcp_url
   resource_packs_gcp_rev = var.resource_packs_gcp_rev
+  append_logs_to_error   = true
```

Comment:
We started to use this in Azure and it proved really valuable for debugging, so also enabling it for the examples and made it configurable for the modules.